docs/setup-cluster/checklists/postgresql-setup-guide.rst

```diff
@@ -0,0 +1,47 @@
+.. _postgresql-database-tuning:
+
+############################
+ PostgreSQL Database Tuning
+############################
+
+When setting up PostgreSQL for your deployment, it's crucial to configure several key parameters to
+ensure optimal performance. Below are the most important settings to consider:
+
+#. **max_connections**: Ensure ``max_connections`` is set to at least 96.
+#. **shared_buffers**: Set ``shared_buffers`` to approximately 25% of total system memory, adjust
+   based on system workloads.
+#. **max_wal_size and min_wal_size**: Adjust ``max_wal_size`` and ``min_wal_size`` according to your
+   usage patterns for improved performance.
+
+For more details, visit the relevant section below.
+
+*****************
+ max_connections
+*****************
+
+``max_connections`` is the most important and universal parameter to adjust. If you are using an
```

Comment:
I would drop "if you are using an existing PostgreSQL installation". Even on a new installation, depending on how you get and install Postgres, the default may be too low.

Reply:
thanks! makes sense.